packages/compiler-sfc/src/parse.ts

```diff
@@ -191,10 +199,12 @@ export function parse(
         const scriptBlock = createBlock(node, source, pad) as SFCScriptBlock
         const isSetup = !!scriptBlock.attrs.setup
         if (isSetup && !descriptor.scriptSetup) {
+          scriptBlock.lang ??= defaultScriptLang
```

Comment:
`??=` is not allowed in Vue, since it's published on [ES2021 Logical Assignment Operators](https://github.com/tc39/proposal-logical-assignment). The lowest compatibility of Vue is ES2015.

See also [Caniuse](https://caniuse.com/mdn-javascript_operators_logical_nullish_assignment)